src/types/Homework/Homework.ts

```diff
@@ -7,4 +8,14 @@ export type Homework = {
   date: Date;
 };
 
+export type HomeworkRequest = {
+  class_id: string | { id: string; value: string };
+  difficulty_id: 1 | 2 | 3;
+  title: string;
+  open_questions: number | undefined;
```

Comment:
I am a bit concerned about using undefined. Shouldn't we stick to numbers and use 0?

Reply:
For our Initial Values of our HomeworkReducer in CreateHomework.tsx, open and closed questions should not be zero as initial values. This is because technically being zero is a valid input (Maybe the teacher doesn't want open questions in its homwork) However, if the teacher never types on the input field, we can't assume it should be zero. Rather, we should tell him that the field is empty, thus the undefined type.